content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -141,6 +141,10 @@ ECS tasks that run on Fargate are already fully allocated [in the CUR][103]. CCM
 
 For Kubernetes data transfer allocation, a Kubernetes node is joined with its associated data transfer costs from the [CUR][103]. The node's cluster name and all node tags are added to the entire data transfer cost for the node. This allows you to associate cluster-level dimensions with the cost of the data transfer, without considering the pods scheduled to the node.
 
+Next, Datadog looks at all of the [workload resources][104] running on that node for the day. The cost of the node is allocated to the workload level based on the resources it has used and the length of time it ran. This calculated cost is enriched with all of the workload resource's tags.
+
+**Note**: Only _tags_ from pods and nodes are added to cost metrics. To include labels, enable labels as tags for [nodes][101] and [pods][102].
+
 [Network Performance Monitoring][105] must be enabled on all AWS hosts to allow accurate data transfer cost allocation. If some hosts do not have Network Performance Monitoring enabled, the data transfer costs for these hosts is not allocated and may appear as an `n/a` bucket depending on filter and group-by conditions.
 
 Datadog supports data transfer cost allocation only through the [standard 6 workload resources][104]. If you use custom workload resources their data transfer costs may only be allocated down to the cluster level and not the node/namespace level.
```

Comment:
should we link to https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/ here?

Reply:
added